sessionize/app/src/main/java/co/touchlab/sessionize/event/EventViewModel.kt

```diff
@@ -3,9 +3,10 @@ package co.touchlab.sessionize.event
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import co.touchlab.sessionize.EventModel
+import co.touchlab.sessionize.ServiceRegistry
 
 class EventViewModel(sessionId:String): ViewModel(){
-    val eventModel = EventModel(sessionId)
+    val eventModel = EventModel(sessionId, ServiceRegistry.analyticsApi, ServiceRegistry.sessionizeApi)
```

Comment:
I think that passing in the services here is not necessary since we have to initialize the service registry in the tests currently anyway. I think this constructor injection is a remanent of me wanting to not do any static initialization in the tests, but since that appears impractical (at least for the purpose of this change) we can stick with just using the service registry directly in the tests (and in EventModel)

Reply:
Changed this back to how it was originally